projects/packages/masterbar/changelog/remove-wpcomsh-phan-stubs-from-masterbar

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: removed
+
+Removed loading wpcomsh phan stubs from the Masterbar, these stubs are already included globally now'
```

Comment:
This doesn't really affect the package at all, just CI checks. So I'd do like this
```suggestion
Significance: patch
Type: removed
Comment: Removed loading wpcomsh phan stubs from the Masterbar, these stubs are already included globally now'


```
That'll then want a `tools/fixup-project-versions.sh` run to bump the versions back down. That might also allow for reverting the version bumps from wpcomsh and mu-wpcom-plugin.